sql/parser/sql.go

```diff
@@ -333,313 +333,319 @@ var yyExca = []int{
 	-2, 0,
```

Comment:
Looks like it isn't possible currently: https://github.com/github/linguist/issues/2206

I'm still looking.